WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/CustomerSearchUICommand.swift

```diff
@@ -86,6 +86,10 @@ final class CustomerSearchUICommand: SearchUICommand {
         featureFlagService.isFeatureFlagEnabled(.betterCustomerSelectionInOrder) && loadResultsWhenSearchTermIsEmpty
     }
 
+    var adjustTableViewBottomInsetWhenKeyboardIsShown: Bool {
+        false
+    }
+
```

Comment:
Since this will be false for the `CustomerSearchUICommand` implentation, we can make it non-computed, right? We just need to keep it as computed in the default `SearchUICommand` implementation for the protocol extension as we can't store properties there.
```suggestion
let adjustTableViewBottomInsetWhenKeyboardIsShown = false
```

Reply:
@iamgabrielma Why can't we use let? I changed it in 86cb87797b679ebe3631b930c05731e9b36bd71f and works fine for me. Thanks for the suggestion!

Reply:
Oh! Interesting, I TIL myself then. I though it should to be `var` for conformance with the protocol, but in this case seems to work anyway.